From: Kovid Goyal <kovid@kovidgoyal.net>
Date: Wed, 22 Dec 2021 21:52:25 +0530
Subject: ssize_t not defined in msvc

---
 src/unrardll/wrapper.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/unrardll/wrapper.cpp b/src/unrardll/wrapper.cpp
index 946ab4d..023943d 100644
--- a/src/unrardll/wrapper.cpp
+++ b/src/unrardll/wrapper.cpp
@@ -123,7 +123,7 @@ static char _process_data[] = "_process_data";
 
 static inline bool
 write_all(const char* data, size_t sz, int fd) {
-    ssize_t written;
+    Py_ssize_t written;
     int err;
     while(sz > 0) {
 #ifdef _WIN32
